aip/0127.md

```diff
@@ -0,0 +1,119 @@
+---
+aip:
+  id: 127
+  state: reviewing
+  created: 2019-08-22
+permalink: /127
+redirect_from:
+  - /0127
+---
+
+# HTTP & gRPC Transcoding
+
+APIs that follow [resource-oriented design][aip-121] are defined as [RPC][]
+APIs, but the resource-oriented design framework allows those APIs to also be
+presented as APIs that largely follow REST/JSON conventions. This is important
+in order to give developers cognitive leverage: over 80% of the public APIs
```

Comment:
That's really "over 80% of the public Google APIs" - we've been trying to be less Google-centric, haven't we? I wonder whether we should just ditch this justification.